conanfile.py

```diff
@@ -256,18 +256,28 @@ def requirements(self):
                 self.requires("xkbcommon/0.8.4@bincrafters/stable")
 
     def system_requirements(self):
+        pack_names = []
+
+        if self.options.qtwebengine:
+            if tools.os_info.is_linux:
+                if tools.os_info.with_apt:
+                    pack_names += ["bison", "flex", "gperf", "python",
```

Comment:
Do you think that bison, flex and gperf system package could be replaced with`build_requires` of `bison-installer/3.4.1@bincrafters/stable`, `flex/2.6.4@bincrafters/stable` and `gperf_installer/3.1@conan/stable`  protected by `if not tools.which('foo'):` like https://github.com/bincrafters/conan-qt/blob/testing/5.13.1/conanfile.py#L152 ?

Reply:
Done, I also add test in test_package.
I still have missing dependencies, so don't merge it yet please.